.github/workflows/main.yml

```diff
@@ -38,3 +38,47 @@ jobs:
           tags: "${{ steps.nscloud.outputs.registry-address }}/nscloud-setup-buildx-action/testdata/server:latest"
           cache-from: "type=registry,ref=${{ steps.nscloud.outputs.registry-address }}/nscloud-setup-buildx-action/testdata/server:buildcache"
           cache-to: "type=registry,ref=${{ steps.nscloud.outputs.registry-address }}/nscloud-setup-buildx-action/testdata/server:buildcache,mode=max"
+
+  build_and_push_published:
```

Comment:
Should we put this "debug output workflow" into a separate workflow which only runs on workflow_dispatch?

Reply:
hmm actually I plan to remove it again becasue I think it's better to perform these tests in `nsc docker buildx setup` on setup time, and automatically generate relevant output if something is bad. WDYT?

Reply:
> Also test on ubuntu-latest
This is still good to do :)